test_launch_ros/test/test_launch_ros/actions/test_node.py

```diff
@@ -219,7 +219,7 @@ def test_launch_node_with_parameter_dict(self):
         file_path, is_file = expanded_parameter_arguments[0]
         assert is_file
         with open(file_path, 'r') as h:
-            expanded_parameters_dict = yaml.load(h, Loader=yaml.FullLoader)
+            expanded_parameters_dict = yaml.safe_load(h, Loader=yaml.FullLoader)
```

Comment:
This change currently fails CI:

> TypeError: safe_load() got an unexpected keyword argument 'Loader'

Reply:
Since CI (https://ci.ros2.org/view/All/job/test_ci_linux/42/pytest-warnings-with-location/) only marks the first case as a warning I went ahead and reverted the second change in ef54e1c7b73e54acfcf3b2339337707d52bd3316.